### PR TITLE
Animate hamburger

### PR DESCRIPTION
**Pull request by udeaghad**
In this pull request, we have:

- created the hamburger button to display the mobile menu when the user clicks it
- the mobile menu disappears when the user clicks on the (x) button
- the mobile menu disappears when the user clicks any menu option
- the correct page is displayed when the user clicks any of the mobile menu options
- updated the major languages used for the project and the contributors on READme.md
- 
Kindly review and recommend changes to implement before we merge to the main branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,50 +10,17 @@
...
-                    <a class="nav-link" href="#contact-me">
-                        Contact
-                    </a>
-                </li>
-            </ul>
-
-             </nav>
-
-            </div>
+                <img class="bar" src="asset/images/Icon-Menu.png" alt="menu Icon">
```
Comment (CollinsTatang) [inline index.html:18]: - [ ] Please kindly fix your hamburger menu, it has to work as follows:

- When the user clicks (or taps) the hamburger button, the mobile menu appears

- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.
harm

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @udeaghad, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (udeaghad) [reply index.html:18]: @CollinsTatang ,

I just checked and the mobile menu hamburger is working well on my computer. I don't know what could be the problem. Please,  can you re-check?

Comment (CollinsTatang) [reply index.html:18]: Hi @udeaghad 
I just check the branch with the pull request the hamburger menu is not working.

**Pull request closed without merging**